cdc/owner.go

```diff
@@ -136,6 +145,10 @@ type Owner struct {
 const (
 	// CDCServiceSafePointID is the ID of CDC service in pd.UpdateServiceGCSafePoint.
 	CDCServiceSafePointID = "ticdc"
+	// CDCMinGCSafePointCacheServiceID is this service GC safe point ID
+	CDCMinGCSafePointCacheServiceID = "ticdc-minGCSafePoint-cache"
```

Comment:
Is adding another safe-point the only way to get the current min safe-point?

Reply:
There is no direct way to get it now.